src/Nox.Solution/NoxSolutionBuilder.cs

```diff
@@ -66,11 +66,19 @@ public NoxSolution Build()
         {
             ResolveAndValidateFilesAndContent();
 
-            if (_yamlFilesAndContent == null) return new NoxSolution();
+            if (_yamlFilesAndContent == null)
+            {
+                var noxSolution = new NoxSolution();
+                // Should we not validate the Default Solution?
```

Comment:
IMO we should explicitly validate the default NoxSolution if it's supposed to be used!